### PR TITLE
reworded comment form

### PR DESCRIPTION
**Pull request by jracusin**
Vidushi and I reworded the questions on the contact form.

### CODE REVIEW

File: app/routes/contact.tsx [hunk 1]
```diff
@@ -99,24 +99,34 @@ export default function () {
         </>
       ) : (
         <Form method="post">
-          <Label htmlFor="service-fieldset">What do you need help with?</Label>
+          <Label htmlFor="service-fieldset">Which version of GCN?</Label>
```
Comment (lpsinger) [inline app/routes/contact.tsx:102]: I prefer the old wording.

"Version" makes me think of version numbers.

File: app/routes/contact.tsx [hunk 2]
```diff
@@ -127,7 +137,7 @@ export default function () {
               setEmailValid(validate(value))
             }}
           />
-          <Label htmlFor="subject">What is your question about?</Label>
```
Comment (lpsinger) [inline app/routes/contact.tsx:130]: I preferred the old wording.

**Review by lpsinger — CHANGES_REQUESTED** [review]
Let's please merge #758 first.

**Comment by jracusin** [comment]
#758 was merged, and conflicts resolved

File: app/routes/contact.tsx [hunk 3]
```diff
@@ -99,35 +99,34 @@ export default function () {
         </>
       ) : (
         <Form method="post">
-          <Label htmlFor="service-fieldset">What do you need help with?</Label>
+          <Label htmlFor="service-fieldset">Which version of GCN?</Label>
```
Comment (lpsinger) [inline app/routes/contact.tsx:102]: This isn't a complete sentence, but it ends with punctuation as if it was.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/contact.tsx (hunk 3 above)
Comment (jracusin) [reply app/routes/contact.tsx:102]: I thought I had reverted this.  Sorry, will fix.

**Comment by jracusin** [comment]
changes have gotten confused as they've gone back and forth.  Will submit another PR.  Do not merge.

**Pull request closed without merging**